src/uproot/reading.py

```diff
@@ -163,6 +163,7 @@ def open(
     "num_fallback_workers": 10,
     "begin_chunk_size": 403,  # the smallest a ROOT file can be
     "minimal_ttree_metadata": True,
+    "range_header_len": 21784,
```

Comment:
Just one thing: since this only applies to HTTP, the option should have `http_` in its name. Also, it's a maximum length, rather than a length (measured in bytes), so maybe

```suggestion
    "http_max_header_bytes": 21784,
```

with appropriate changes to pick up the name modification elsewhere?